source/image-handler/lambda_function.py

```diff
@@ -213,7 +213,7 @@ def request_thumbor(original_request, session):
 
 def process_thumbor_responde(thumbor_response, vary):
      if thumbor_response.status_code != 200:
-         return response_formater(status_code=response.status_code)
+         return response_formater(status_code=thumbor_response.status_code)
```

Comment:
I believe using `response` here was a mistake; at one point when running my lambda function when I had an issue with `tc_aws` it hit this code path and blew up saying that `response` wasn't a global.

I believe `thumbor_response` is what was intended here.

Reply:
Looks like this was fixed in https://github.com/awslabs/serverless-image-handler/pull/34 as well